src/main/java/com/codeborne/selenide/ConditionWaiter.java

```diff
@@ -0,0 +1,42 @@
+package com.codeborne.selenide;
+
+import org.awaitility.core.ConditionTimeoutException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.function.Predicate;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.awaitility.Awaitility.await;
+
+
+public class ConditionWaiter {
+
+  private static final Logger logger = LoggerFactory.getLogger(ConditionWaiter.class);
+
+  public static void waitFor(Conditional conditional, Predicate predicate, String message) {
+    waitFor(conditional, predicate, message, 10);
+  }
+
+  public static void waitFor(Conditional conditional, Predicate predicate, String message, int timeout) {
+    try {
+      await()
+        .pollInterval(1, SECONDS)
```

Comment:
In any way this case should be configurable.
You can reload method with default value, but add an ability to set this parameter with custom value as well as `timeout`.